soc/arm/atmel_sam/same70/dts_fixup.h

```diff
@@ -127,7 +127,7 @@
 #define DT_USBHS_PERIPHERAL_ID	DT_ATMEL_SAM_USBHS_40038000_PERIPHERAL_ID
 #define DT_USBHS_RAM_BASE_ADDRESS	DT_ATMEL_SAM_USBHS_40038000_RAM_0
 
-#define CONFIG_ENTROPY_NAME			DT_ATMEL_SAM_TRNG_40070000_LABEL
+#define DT_ENTROPY_NAME			DT_ATMEL_SAM_TRNG_40070000_LABEL
```

Comment:
Why isn't this `DT_LABEL(DT_INST(0, atmel_sam_trng))`?

Reply:
just historic, leaving what that part of the define alone.